pygmt/base_plotting.py

```diff
@@ -115,6 +115,9 @@ def coast(self, **kwargs):
             ``'river[/pen]'``
             Draw rivers. Specify the type of rivers and [optionally] append pen
             attributes.
+        L : str
+            ``'[g|j|J|n|x]refpoint'``
+            Draws a simple map scale centered on the reference point specified.            
```

Comment:
```suggestion
            Draws a simple map scale centered on the reference point specified.
```
Just remove the extra whitespace to make the tests pass.